.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+    "extends": "./node_modules/@biotope/quality-gate/config/logic/recommended.js",
+    "rules": {
+        "no-console": "off",
+        "compat/compat": "off",
+        "import/no-dynamic-require": "off",
+        "global-require": "off",
+        "@typescript-eslint/no-var-requires": "off",
+        "no-template-curly-in-string": "off",
+        "import/no-cycle": "off",
+        "@typescript-eslint/no-explicit-any": "off"
```

Comment:
Please check which rules you think should definitely be enabled @SheepFromHeaven